False positive no-member errors.

**TODO:** make sure to post the output of `pip freeze`
appnope==0.1.0
astroid==2.0.4
backcall==0.1.0
decorator==4.3.0
Django==2.1
django-extensions==2.1.0
django-guardian==1.4.9
djangorestframework==3.8.2
drf-nested-routers==0.90.2
dry-rest-permissions==0.1.10
ipython-genutils==0.2.0
isort==4.3.4
jedi==0.12.0
lazy-object-proxy==1.3.1
mccabe==0.6.1
parso==0.2.1
pexpect==4.6.0
pickleshare==0.7.4
pipfile==0.0.2
prompt-toolkit==1.0.15
psycopg2==2.7.5
ptyprocess==0.5.2
Pygments==2.2.0
pylint==2.1.1
pylint-django==2.0.1
pylint-plugin-utils==0.4
pytz==2018.5
simplegeneric==0.8.1
six==1.11.0
toml==0.9.4
traitlets==4.3.2
typed-ast==1.1.0
uWSGI==2.0.17.1
wcwidth==0.1.7
wrapt==1.10.11
yapf==0.22.0


1.  When I try to remove a user from a group using the following expression:
`self.user.groups.remove(self.some_group)`

I get the following error:
`api/models.py:451:12: E1101: Instance of 'QuerySet' has no 'remove' member (no-member)
api/models.py:451:12: E1101: Instance of 'ManyToManyField' has no 'remove' member (no-member)`

2.  When I try to change a ManyToManyField using .set like this:
`self.some_many_to_many_field.set(userlist)`
 
I get the following error:
`api/models.py:525:8: E1101: Instance of 'QuerySet' has no 'set' member; maybe 'get'? (no-member)
api/models.py:525:8: E1101: Instance of 'ManyToManyField' has no 'set' member (no-member)`

--- a/pylint_django/augmentations/__init__.py
+++ b/pylint_django/augmentations/__init__.py
@@ -248,6 +248,8 @@ MANYTOMANY_FIELD_ATTRS = {
     'clear',
     'related_name',
     'related_query_name',
+    'remove',
+    'set',
     'limit_choices_to',
     'symmetrical',
     'through',

--- a/pylint_django/tests/input/func_noerror_manytomanyfield.py
+++ b/pylint_django/tests/input/func_noerror_manytomanyfield.py
@@ -45,6 +45,15 @@ class CustomUser(AbstractUser):  # pylint: disable=model-no-explicit-unicode
             self.user_permissions.add(perm)
         return self.user_permissions
 
+    def add_permission(self, permission):
+        self.user_permissions.add(permission)
+
+    def remove_permission(self, permission):
+        self.user_permissions.remove(permission)
+
+    def set_permissions(self, permissions):
+        self.user_permissions.set(permissions)
+
     def save(self, *args, **kwargs):
         ''' Saving while granting new permissions '''
         self.is_staff = True

